github/ci/prow-deploy/files/jobs/kubevirt/common-instancetypes/common-instancetypes-presubmits.yaml

```
@@ -695,6 +695,192 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - name: pull-common-instancetypes-kubevirt-debian-11
+    branches:
+      - main
+    always_run: false
+    run_if_changed: "preferences/linux-efi/.*|preferences/linux/.*|preferences/base/.*|preferences/components/.*|preferences/debian/.*|tests/functests/.*|tests/vendor/.*|tests/go.*"
```

```suggestion
    run_if_changed: "preferences/base/.*|preferences/components/.*|preferences/linux/.*|preferences/linux-efi/.*|preferences/debian/.*|tests/functests/.*|tests/vendor/.*|tests/go.*"
```

Can you order it like this? Also, depending on which shared prefs other lanes use, they need to be updated as well.

---

Done. Currently, only debian preference uses `linux` and `linux-efi`, I plan to update the rest of the preferences to use `linux` and `linux-efi` as follow-up, I will update the test jobs then.